web/html/src/manager/content-management/shared/components/panels/environment-lifecycle/environment-lifecycle.js

```diff
@@ -8,17 +8,20 @@ import Promote from "../promote/promote";
 import {showErrorToastr, showSuccessToastr} from "components/toastr/toastr";
 import {mapAddEnvironmentRequest, mapUpdateEnvironmentRequest} from './environment.utils';
 
-import type {ProjectEnvironmentType} from '../../../type/project.type.js';
-import type {ProjectHistoryEntry} from "../../../type/project.type";
 import useRoles from "core/auth/use-roles";
 import {isOrgAdmin} from "core/auth/auth.utils";
 import useLifecycleActionsApi from "../../../api/use-lifecycle-actions-api";
+import getRenderedMessages from "../../messages/messages";
+
+import type {ProjectEnvironmentType, ProjectHistoryEntry, ProjectMessageType}
+  from "../../../type/project.type";
```

Comment:
I prefer having one line per type imported... but that is cosmetic

Reply:
I'd better do everything in one line instead. If the source file is moved to some other place, we need to adjust only one line instead of many. But this is still cosmetic and it's just my 2c. :wink:

Reply:
Thanks for the opinions, since there's no consensus here, I'll keep it like this :angel: